### ISSUE TITLE
i3d_32frames_85.63.checkpoint 

### ISSUE THREAD

**Issue by asdasdfasdfsd** [anita-hu/MSAF#6]
Hi. Thanks for sharing your work.
May I ask how you got the i3d_32frames_85.63.checkpoint?

When I downloaded i3d checkpoint pre-trained on imagenet/kinetics from https://github.com/rimchang/kinetics-i3d-Pytorch/tree/master/data/pytorch_checkpoints and fine-tuned the network on NTU DB, the accuracy was about  82.56. 

And also when I downloaed  i3d_32frames_85.63.checkpoint from your repo and fine-tuned the I3D network on NTU, the accuracy wen up to 88.41.

Thank you.

**Comment by anita-hu** [comment]
Hi, 
We also used the pre-trained model `rgb_imagenet.pkl` from the repository you linked. We fine-tuned the model on NTU RGB D using `torch.optim.SGD(model.parameters(), lr=0.01, momentum=0.9)` with batch size 4 and the best model was at 18 epochs. 

Out of curiosity, what configuration did you use to get 88.41?

**Comment by asdasdfasdfsd** [comment]
Thank you for reply.
I load i3d_32frames_85.63.checkpoint and fine-tuned the model on NTU using following configurations.
optimizer = torch.optim.Adam(model.parameters(), lr=.0001, weight_decay=1e-4), 
update_lr(optimizer, multiplier=.1), when training epoch is 10 and 20
The bactch size is set to 48 and I trained the network for 40 epochs. 
Thank you.
